blocksuite/affine/data-view/src/core/group-by/trait.ts

```
@@ -96,48 +96,44 @@ export class GroupTrait {
     const propertyId = this.propertyId;
     if (!propertyId) {
       return;
     }
-    this.view.propertyDataSet(propertyId, data);
+    this.view.propertyGetOrCreate(propertyId).dataUpdate(() => data);
   };
 
   get addGroup() {
     const type = this.property$.value?.type$.value;
     if (!type) {
       return;
```

Biome (1.9.4)

[error] This return should return a value because it is located in a getter.



(lint/suspicious/useGetterReturn)